src/gc/gc.d

```diff
@@ -1476,7 +1719,7 @@ struct Gcx
     /**
      *
      */
-    void removeRoot(void *p) nothrow @nogc
+    void removeRoot(void *p) nothrow
```

Comment:
It must have gotten removed on accident, which is strange because I saw the PR that added them in and I was trying to avoid them getting taken out again by mine.